host/pages/_app.js

```diff
@@ -2,17 +2,14 @@ import Head from "next/head";
 import "../styles/globals.css";
 import React from "react";
 
-//This is a part of a hack I need (sorry for that). Take a look at next.config.js
-globalThis.React = React;
+
+const SmartButton = (await import ("remoteLib/SmartButton")).default;
 
 function MyApp({ Component, pageProps }) {
   return (
-    <>
-      <Head>
-        <script src="http://localhost:3002/web/remoteEntry.js" />
-      </Head>
-      <Component {...pageProps} />
-    </>
+    <div>
```

Comment:
Doesn't this break the nextjs functionality? Might just be better to remove the head + still use http://localhost:3000/testing